multinet/uploaders/swagger/chunk_upload.yaml

```diff
@@ -0,0 +1,41 @@
+Upload one chunk of data to the corresponding `uploads` document
+---
+parameters:
+  - $ref: "#/parameters/upload_id"
+  - name: sequence
+    in: query
+    description: The sequence number of the multipart upload
+    required: true
+    schema:
+      type: number
+      example: 0
```

Comment:
There's a missing parameter--the chunk data itself. See `aql.yaml` for an example of how to specify a body parameter in Swagger.